### PR TITLE
change default from machine-id to @saved

### PR DESCRIPTION
**Pull request by Pogogo007**

### CODE REVIEW

File: src/modules/bootloader/main.py [hunk 1]
```diff
@@ -255,7 +256,7 @@ def create_loader(loader_path, installation_root_path):
         loader_entries = []
         pass
 
-    lines = [f"default {machine_id}*"]
+    lines = [f"default @saved"]
```
Comment (vnepogodin) [inline src/modules/bootloader/main.py]: Remove f-string

**Review by vnepogodin — CHANGES_REQUESTED** [review]

File: src/modules/bootloader/main.py [hunk 2]
```diff
@@ -244,9 +244,10 @@ def create_loader(loader_path, installation_root_path):
     :param installation_root_path: The path to the root of the target installation
     """
 
+    #Obsolete since default was changed to @saved from machine-id
```
Comment (SoulHarsh007) [inline src/modules/bootloader/main.py]: Add a space after `#`

File: src/modules/bootloader/main.py (hunk 1 above)
Comment (Pogogo007) [reply src/modules/bootloader/main.py]: Forgot python uses that. Thanks

File: src/modules/bootloader/main.py (hunk 2 above)
Comment (Pogogo007) [reply src/modules/bootloader/main.py]: Changed to using a multi line comment. Thanks

**Review by vnepogodin — APPROVED** [review]
LGTM

**Pull request merged**